lib/Service/OpenAiSettingsService.php

```diff
@@ -343,12 +351,7 @@ public function setQuotas(array $quotas): void {
 	 */
 	public function setAdminApiKey(string $apiKey): void {
 		// No need to validate. As long as it's a string, we're happy campers
-		if ($apiKey === '') {
-			$this->config->setAppValue(Application::APP_ID, 'api_key', '');
-		} else {
-			$encryptedApiKey = $this->crypto->encrypt($apiKey);
-			$this->config->setAppValue(Application::APP_ID, 'api_key', $encryptedApiKey);
-		}
+		$this->appConfig->setValueString(Application::APP_ID, 'api_key', $apiKey, false, true);
```

Comment:
this needs to be encrypted probably

Reply:
the 4th param of appConfig->setValueString is "sensitive". Setting it to true encrypts the value.

Reply:
saw the PR description again, and very cool!